v2/emailpassword/graphql-integration/backend-setup.mdx

```diff
@@ -32,69 +36,186 @@ Follow the [frontend and backend custom UI setup guides](../custom-ui/init/front
 
 ## b) Install GraphQL dependencies on your backend
 
-:::note
-We use express-graphql as an example, SuperTokens can be used with other GraphQL libraries.
+:::info
+We use Apollo server in this example, SuperTokens can be used with other GraphQL libraries as well.
 :::
 
 ```bash
-npm i express-graphql graphql
+npm install @apollo/server graphql
 ```
 
 ## c) Setup GraphQL
 
-### Create a schema 
+### Create your schema 
 
-```tsx
-import { buildSchema } from 'graphql';
+In this example we will create a simple social app that allows logged in users to create and view posts. To build an app like this you would need the following:
+- A way for users to login and logout
```

Comment:
not required for non session recipes?

Reply:
We want to demonstrate that you would need the following functionality, but in the next step we explain that we dont add any code for it because the SDK already exposes endpoints for it